src/client/css/partials/breaches.css

```diff
@@ -252,3 +256,41 @@
 .breaches-table details[open] > article {
   animation: fade-in 0.5s ease-out;
 }
+
+.breaches-table .no-unresolved-breaches-message > .no-breaches-message,
+.breaches-table .no-unresolved-breaches-message > .all-breaches-resolved-message {
+  display: none;
+}
+
+.breaches-table[data-nr-unresolved='0'][data-nr-resolved='0'] .no-unresolved-breaches-message .no-breaches-message {
+  display: flex;
+}
+
+.breaches-filter:not([data-selected='resolved']) + .breaches-table[data-nr-unresolved='0']:not([data-nr-resolved='0']) .no-unresolved-breaches-message .all-breaches-resolved-message {
+  display: flex;
+}
+
+.no-breaches-message,
+.all-breaches-resolved-message {
+  flex-direction: column;
+  align-items: center;
+  gap: var(--padding-sm);
+  padding: var(--padding-xl) var(--padding-md);
+}
+
+.no-breaches-message h2,
+.all-breaches-resolved-message h2 {
+  font-weight: 600;
+}
+
+.no-breaches-message p,
+.all-breaches-resolved-message p {
+  font: var(--h3-font);
+  font-weight: 500;
+  max-width: 700px;
+}
+
+.no-breaches-message p b,
+.all-breaches-resolved-message p b {
+  font-weight: 700;
```

Comment:
These and the font settings for the `p`s above don't seem to have any effect. I expect that the (Metropolis) fonts aren't initialised for these weights? Was that a conscious decision?

Reply:
Only Metropolis Bold `700` is initialized in `src/client/css/fonts.css`. I guess that’s probably just due to the zero state using it for the first time?

Reply:
Ah gotcha. I added them here, of course at the cost of two additional resources to fetch: 97351dbf

Reply:
Ah, I was late to this party.  Yes it was a conscious decision – we agreed with UX that we would try to conform to a limited set of fonts to keep it simple and avoid extra client loads.

Same with heading and paragraph sizes – let's conform to the closest established variable and get UX sign-off.

For this case since it was already merged, I can chat with UX and post a follow-up PR with any adjustments.